metadata-integration/java/spark-lineage/src/main/java/datahub/spark/DatasetExtractor.java

```diff
@@ -29,55 +31,50 @@
 import org.apache.spark.sql.sources.BaseRelation;
 
 import com.google.common.collect.ImmutableSet;
+import com.linkedin.common.FabricType;
 
 import scala.Option;
 import scala.collection.JavaConversions;
 
+import com.typesafe.config.Config;
+
 public class DatasetExtractor {
   private static final Map<Class<? extends LogicalPlan>, PlanToDataset> PLAN_TO_DATASET = new HashMap<>();
   private static final Map<Class<? extends BaseRelation>, RelationToDataset> REL_TO_DATASET = new HashMap<>();
-  private static final Set<Class<? extends LogicalPlan>> OUTPUT_CMD = ImmutableSet
-      .of(InsertIntoHadoopFsRelationCommand.class, SaveIntoDataSourceCommand.class,
-          CreateDataSourceTableAsSelectCommand.class, CreateHiveTableAsSelectCommand.class,
-          InsertIntoHiveTable.class);
+  private static final Set<Class<? extends LogicalPlan>> OUTPUT_CMD = ImmutableSet.of(
+      InsertIntoHadoopFsRelationCommand.class, SaveIntoDataSourceCommand.class,
+      CreateDataSourceTableAsSelectCommand.class, CreateHiveTableAsSelectCommand.class, InsertIntoHiveTable.class);
+  private static final String DATASET_ENV_KEY = "metadata.dataset.env";
+  private static final String DATSET_PLATFORM_INSTANCE_KEY = "metadata.dataset.platform_instance";
```

Comment:
suggest you use camel case here to align with Java / Scala style.. 
so `metadata.dataset.platformInstance` as the config key.

Reply:
Done